libs/navigation/build.mjs

```diff
@@ -0,0 +1,20 @@
+import * as esbuild from 'esbuild'; // eslint-disable-line
```

Comment:
Just FYI, we also use `microbundle` for things such as creating the preact-htm bundle